source/reference/limitations.txt

```diff
@@ -150,6 +150,9 @@ Sharded Clusters
   returns an error and stops, which requires you to start the migration
   from scratch.
 
+- :dbcommand:`transitionFromDedicatedConfigServer` is not supported during 
```

Comment:
[nit/suggestion] Can we make this more active voice? Maybe something like: 
```
You can't run the :dbcommand:`transitionFromDedicatedConfigServer` command while...
```
or 
```
``mongosync`` doesn't support running the :dbcommand:`transitionFromDedicatedConfigServer` during...
```